packages/main/src/Table.ts

```diff
@@ -507,6 +515,10 @@ class Table extends UI5Element {
 		return this.shadowRoot!.getElementById("table") as HTMLElement;
 	}
 
+	get _loadingElement() {
+		return this.shadowRoot!.getElementById("loading") as HTMLElement;
```

Comment:
I would suggest to switch to querySelector because it accepts generic type where you can do it in following way:
```ts
this.shadowRoot!.querySelector<BusyIndicator>("loading");
```

With that we will keep the original type `BusyIndicator | undefined` because the element is conditionally rendered inside the shadow DOM.

Reply:
hi @nnaydenow thanks for your comment. I see your point but this is an internal accessor where we know that the instance must exist when it is called and we use only its focus API where it inherits from HTMLElement. 
As you know at the end this line will be transpired to `this.shadowRoot.getElementById("loading")` .

We have too many _(after|before|grid|growing|loading)Element API and we are considering to change those to a generic e.g. $(selector) API so I do not want to change this single here.

Reply:
The other reason why I do not want to change it is that I had to trigger voters 4 times to get +1 from them :(
https://github.com/SAP/ui5-webcomponents/actions/runs/9501544420